esp-hal/build.rs

```diff
@@ -62,9 +62,21 @@ fn main() -> Result<(), Box<dyn Error>> {
     println!("cargo:rerun-if-changed=./esp_config.yml");
     let cfg_yaml = std::fs::read_to_string("./esp_config.yml")
         .expect("Failed to read esp_config.yml for esp-hal");
-    let cfg =
+    let mut cfg =
         generate_config_from_yaml_definition(&cfg_yaml, true, true, Some(config.clone())).unwrap();
 
+    // esp-rom-sys might wish to contribute to .rwtext
+    cfg.insert(
+        "RWTEXT_ADDITION".to_string(),
```

Comment:
This feels a bit flaky. We're passing linker directives via a dummy config?

Reply:
ah yes - I thought of the way how we passed it from esp-rom-sys to esp-hal

Yes, here we used "a well defined special config" (aka dummy config). Won't show up in the config table since that is done before in `generate_config_from_yaml_definition` .... anyways it's gone for now 😄